### DISCUSSION TITLE
What features should daedalOS focus on in 2023?

### DISCUSSION THREAD

**Discussion by DustinBrett** [DustinBrett/daedalOS#234] [Polls]
As 2023 gets closer and my Post-it wall gets bigger, I wonder what features to focus on for daedalOS in the coming year. I wanted to start a discussion to see what everyone thinks is worth adding/coding next.

**Comment by sethleedy** [comment]
A friend and I have been looking for something like this as a front end for development purposes. Would love to be able to create our own packages and go from there.

**Reply by DustinBrett** [reply]
Cool! It's for sure something I want to make easier for people to do. Until then there is still the option to fork this repo and manually add apps, similar to how the existing ones are added.
